Orbit.h

```diff
@@ -224,15 +229,29 @@ inline void Stop() {
 }
 
 inline void StartAsync(const char* name, uint64_t id, orbit::Color color) {
```

Comment:
"id" is uint32_t in other signatures (also in the one you are calling a few lines below).

Reply:
Yes, thanks, fixed.